lib/active_merchant/billing/gateways/litle.rb

```diff
@@ -62,19 +72,24 @@ def credit(money, authorization, options = {})
         refund(money, authorization, options)
       end
 
-      def refund(money, authorization, options={})
-        transaction_id, _, _ = split_authorization(authorization)
-
+      def refund(money, payment, options={})
         request = build_xml_request do |doc|
           add_authentication(doc)
           add_descriptor(doc, options)
-          doc.credit(transaction_attributes(options)) do
-            doc.litleTxnId(transaction_id)
-            doc.amount(money) if money
+          doc.send(refund_type(payment), transaction_attributes(options)) do
+            if payment.is_a?(String) || payment.is_a?(Numeric)
```

Comment:
Why the need to check for a numeric type now?

Reply:
@dtykocki in the tests it was giving tokens/auth codes as numbers and not strings

Reply:
Shouldn't `authorization_from` stringify them?

Reply:
Yeah `authorization_from` should turn them into strings so I'm a bit 😕 here.

Reply:
@dtykocki and @curiousepic So I can change remote tests so they pass only strings and not numbers. Cases like this: https://github.com/activemerchant/active_merchant/blob/9885a343a0bd9a004b3a3cff9bb2f34357f53ad3/test/remote/gateways/remote_litle_test.rb#L320

Reply:
@nfarve - Try that and maybe we'll be able to remove that number check.